deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -93,12 +93,13 @@ Resources:
               condition: START
           image: >-
             611364707713.dkr.ecr.us-west-2.amazonaws.com/amazon/aws-otel-goxray-sample-app:v0.0.1
-          essential: true
+          essential: false
```

Comment:
In this template/example, we want to run otel collector together with sample apps. is it our expectation to put the sample app `essential` to false?

Reply:
One of the reason is that the container receiver should be able to collect the stopped container's metadata. We don't want to make an example that when the customer use AOC, they should make their container essential, because even one of the containers dead, the task shouldn't be dead.